### DISCUSSION TITLE
Inverse Measurable habit

### DISCUSSION THREAD

**Discussion by saurabhs** [iSoron/uhabits#1019] [Feature Requests] [answered]
Currently the measurable habits have a target to hit (like, read x no. of pages per day/week), can there be another type of the habit which does the opposite, like browse no more than x hrs per day/week?

**Comment by DeWarner** [comment]
this would also be good for having targets for when you get out of bed etc.

**Comment by hiqua** [comment, marked as answer]
Done already, but not available yet: https://github.com/iSoron/uhabits/pull/1101

**Comment by herrbrixter** [comment]
Related to #696
